### PR TITLE
[ao] updating embedding_bag support for fx and eager

### PR DESCRIPTION
**Pull request by HDCharles**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #103467
* __->__ #107623

Summary: our docs were saying dynamic embedding bag wasn't supported but
it actually is (at least at the same level as embeddings were) it just wasn't previously tested/listed.

Test Plan: python test/test_quantization.py -k "test_embedding"

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D51489344](https://our.internmc.facebook.com/intern/diff/D51489344)

### CODE REVIEW

File: docs/source/quantization.rst [hunk 1]
```diff
@@ -129,7 +128,7 @@ Note that for FX quantization, the corresponding functionals are also supported.
 |nn.EmbeddingBag            | Y (activations    |                    |
 |                           | are in fp32)      | Y                  |
 +---------------------------+-------------------+--------------------+
-|nn.Embedding               | Y                 | N                  |
+|nn.Embedding               | Y                 | Y                  |
```
Comment (jerryzh168) [inline docs/source/quantization.rst:131]: could you add ("float qparams weight only") in the Dynamic column:

```
Y (float qparams weight only)
```

File: docs/source/quantization.rst [hunk 2]
```diff
@@ -810,7 +809,7 @@ Note that for FX Graph Mode Quantization, the corresponding functionals are also
...
```
Comment (jerryzh168) [inline docs/source/quantization.rst:812]: same here

**Review by jerryzh168 — APPROVED** [review]

File: docs/source/quantization.rst (hunk 1 above)
Comment (HDCharles) [reply docs/source/quantization.rst:131]: isn't it fp32 activation for both embedding and embedding bag for both static and dynamic? It seems like it should be Y (float qparams weight only) for all 4 entries for both fx and eager mode.

**Comment by HDCharles** [comment]
@HDCharles has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D51489344).

**Pull request closed without merging**